x-pack/test/functional/apps/lens/smokescreen.ts

```diff
@@ -66,18 +66,21 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       // legend item(s), so we're using a class selector here.
       expect(await find.allByCssSelector('.echLegendItem')).to.have.length(3);
     });
+
     it('should create an xy visualization with filters aggregation', async () => {
```

Comment:
In my opinion this function test is covering the simplest case, and I think I've covered the other cases in jest-based tests